zebrad/src/components/inbound.rs

```diff
@@ -21,6 +21,7 @@ use zebra_consensus::transaction;
 use zebra_consensus::{chain::VerifyChainError, error::TransactionError};
 use zebra_network::AddressBook;
 
+use super::mempool as mp;
```

Comment:
Optional: can probably be joined with the import below.

Reply:
Done in 889c9549